xml/System.Reflection/Binder.xml

```diff
@@ -147,7 +147,7 @@
         <param name="value">The field value used to locate a matching field.</param>
         <param name="culture">An instance of <see cref="T:System.Globalization.CultureInfo" /> that is used to control the coercion of data types, in binder implementations that coerce types. If <paramref name="culture" /> is <see langword="null" />, the <see cref="T:System.Globalization.CultureInfo" /> for the current thread is used.  
   
- Note   For example, if a binder implementation allows coercion of string values to numeric types, this parameter is necessary to convert a <see langword="String" /> that represents 1000 to a <see langword="Double" /> value, because 1000 is represented differently by different cultures. The default binder does not do such string coercions.</param>
+ Note   For example, if a binder implementation allows coercion of string values to numeric types, this parameter is necessary to convert a <see langword="String" /> that represents 1000 to a <see cref="T:System.Double" /> value, because 1000 is represented differently by different cultures. The default binder does not do such string coercions.</param>
```

Comment:
Can you please put these notes in the remarks?

Reply:
Other similar notes (search `1000` in `File changed`) are placed in `<param>`s. What to do with them?

Reply:
Feel free to move them if you consider those extra explanations do not provide an essential description of the param, and instaead they provide additional information that does not affect the understanding of the param, then it should go in remarks. Remember that all this documentation (except remarks) is also going to show up in Visual Studio's IntelliSense.
Let me give you a suggestion for this one, so you know the required structure:
```suggestion
```

Reply:
I can't give you a suggestion in the remarks below due to the GitHub restriction. But those remarks would look like this:

```xml
<remarks>
          <format type="text/markdown"><![CDATA[  
  
## Remarks  
 If `bindingAttr` does not include <xref:System.Reflection.BindingFlags.SetField?displayProperty=nameWithType>, the default binder implementation provided by <xref:System.Type.DefaultBinder%2A?displayProperty=nameWithType> simply returns the first element of `match`. No selection is done.  
  
 This method controls the binding provided by <xref:System.Type.InvokeMember%2A?displayProperty=nameWithType>.  
  
If a binder implementation allows coercion of string values to numeric types, the `culture` parameter is necessary to convert a string that represents 1000 to a double value, because 1000 is represented differently by different cultures. The default binder does not do such string coercions.

 ]]></format>
        </remarks>
```